pyp2rpm/bin.py

```diff
@@ -246,7 +247,14 @@ def main(package, v, prerelease, d, s, r, proxy, srpm, p, b, o, t, venv, autonc,
 
         if srpm:
             msg = utils.build_srpm(spec_path, d)
-            logger.info(msg)
+            if isinstance(msg, bytes):
+                for line in msg.decode(locale.getpreferredencoding(),
```

Comment:
Why use locale.getpreferredencoding instead of just 'utf-8'? I assume that on systems that don't use a UTF-8 locale all sorts of other things will go wrong too.

Reply:
locale.getpreferredencoding() is consistent with other uses of decode() in the application (as in archive.py and package_data.py), and will return the encoding indicated by LANG or LC_CTYPE.  Since we're decoding the output of the "rpmbuild" command, I would expect its output to be in the encoding specified by one of those variables.

Reply:
Fair enough. then LGTM